src/HTMLGenerator/converter.py

```diff
@@ -60,3 +61,57 @@ def convert(top_level_output_directory, top_level_input_directory, json_with_fil
     os.remove(current_configuration_absolute_path)
 
     return
+
+def convertFromCli(jsonFilePath) :
```

Comment:
Why is most of the code from the other function copy-pasted here? You should be able to reuse the whole thing.

Reply:
I will refactor now . I wanted to make sure it works correctly first.